Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -494,7 +494,7 @@ void WebLoaderStrategy::scheduleLoadFromNetworkProcess(ResourceLoader& resourceL
     loadParameters.isDisplayingInitialEmptyDocument = frame && frame->loader().stateMachine().isDisplayingInitialEmptyDocument();
     if (frame)
         loadParameters.effectiveSandboxFlags = frame->loader().effectiveSandboxFlags();
-    if (auto openerFrame = frame ? frame->loader().opener() : nullptr) {
+    if (auto* openerFrame = frame ? dynamicDowncast<LocalFrame>(frame->loader().opener()) : nullptr) {
```

Comment:
This can easily be put into a separate PR that obviously doesn't change behavior.

Reply:
👍 https://github.com/WebKit/WebKit/pull/12989